.golangci.yml

```diff
@@ -2,7 +2,6 @@ run:
   deadline: 5m
 
 linters:
-  disable-all: true
   enable:
```

Comment:
Please, split the modifications to `.golangci.yml` to a separated commit (maybe PR), since it is unrelated to how golangci-lint is installed.